### ISSUE TITLE
Accessibility: unable to lock a recording when voice-over is turned on

### ISSUE THREAD

**Issue by Falcopus1986** [signalapp/Signal-iOS#4746]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.
It is not possible to lock the recording when making a voice message when voice-over is turned on. On other platforms you can click on the “record” button and swipe your finger up to lock the recording and you don’t have to keep your finger on the screen all the time. Since speech is the only way for me to read the screen it is difficult for me to find the “cancel” and “send” buttons when the recording is locked and voice-over does not work at that point. In other messaging apps the “record” button changes in a “send” button and Voice-Over keeps working while recording your message. So it is possible to do this trick.

### Steps to reproduce
- using hyphens as bullet points
- list the steps
- that reproduce the bug

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behaviour)
I can not use the lock recording feature when Voice-Over is turned on.
**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behaviour)
Voice-Over should be able to tell where the "send" and "cancel" buttons are when you are recording a voice message.
### Device info

**Device**: iDevice XS

**iOS version**: 14.3.0

**Signal version:** 5.2.0.5

**Comment by FelixGruetzmacher** [comment]
I can reproduce. Came here to report this and found it was already reported.

**Comment by MarcoZehe** [comment]
Also confirmed, and adding my voice.

**Comment by FelixGruetzmacher** [comment]
So has this been fixed?

Am Mo., 31. Jan. 2022 um 03:31 Uhr schrieb stale[bot] <
***@***.***>:
